### ISSUE TITLE
Update-check package is throwing an error

### ISSUE THREAD

**Issue by mofodox** [zeit/next.js#4226]
Hi there,

I am currently experiencing an error caused by update-check. Whenever I run the command `npm run dev` it throws me this error: [error_image](https://cl.ly/3A2T37353S1A)

But why is it that when I run `npm run start` which it started the server without any error.

Hope to hear from the team soon! Thank you!

My Environment
| Tech    | Version |
|---------|---------|
| next    |    6.0.0     |
| node    |   10.0.0      |
| OS      |     MacOS High Sierra 10.13.4 (17E199)    |
| browser |    Google Chrome Version 66.0.3359.139 (Official Build) (64-bit)   |

**Comment by timneutkens** [comment]
No clue, maybe you were running it on an older version of Node.js 🤔

**Comment by brainkim** [comment]
Also running into this error. At first I thought it was because of something to do with my 6-7 babel upgrade but today I checked and it’s happening on fresh installs as well.
https://github.com/zeit/update-check/issues/3

```
npm run dev

> poop@ dev /Users/brian/Projects/poop
> next

/Users/brian/Projects/poop/node_modules/update-check/index.js:18
const getFile = async (details, distTag) => {
                      ^
SyntaxError: Unexpected token (
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Server._callee$ (/Users/brian/Projects/poop/node_modules/next/dist/server/index.js:194:34)
    at tryCatch (/Users/brian/Projects/poop/node_modules/regenerator-runtime/runtime.js:62:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/brian/Projects/poop/node_modules/regenerator-runtime/runtime.js:296:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/brian/Projects/poop/node_modules/regenerator-runtime/runtime.js:114:21)
    at step (/Users/brian/Projects/poop/node_modules/@babel/runtime/helpers/asyncToGenerator.js:12:30)
    at _next (/Users/brian/Projects/poop/node_modules/@babel/runtime/helpers/asyncToGenerator.js:27:9)
    at /Users/brian/Projects/poop/node_modules/@babel/runtime/helpers/asyncToGenerator.js:34:7
```
This is using node version 6.11 and 6.12.

**Comment by yestoall** [comment]
i have the same problem too
any ideas?

**Comment by brainkim** [comment]
@yestoall fixed it by adding "next/babel" to a .babelrc file. This isn’t mentioned in the installation notes nor was it required in 5 but it works now 🤷‍♂️

**Comment by uuau99999** [comment]
@brainkim  I have added next/babel to .babelrc but the issue still exists.

**Comment by brainkim** [comment]
@uuau99999 The issues was the node version not the babel config. Upgrading to node.js 8 fixed it and I got confused! Sorry.

**Comment by timneutkens** [comment]
Also see #4283, it's released as next@canary right now, but will be 5.0.1 soon.

**Comment by uuau99999** [comment]
@brainkim Yup, when I switch to node.js 8.x the problem disappeared. It seems that the problem is fixed because of the native support of async/await in node.js 8.x.  

@timneutkens I will try that later. Thanks.
